task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskCommentsDrawer.vue

```diff
@@ -135,7 +146,19 @@
     },
     mounted() {
       this.$root.$on('displayTaskComment', taskCommentDrawer => {
-        this.showTaskCommentDrawer = true
+        this.showTaskCommentDrawer = true;
+      });
+
+      this.$root.$on('displaySubCommentEditor', id => {
+        if( id == null ) {
+          window.setTimeout(() => {
+            this.openEditorToBottom();
+          }, 500);
+        } else {
+          window.setTimeout(() => {
+            document.dispatchEvent(new CustomEvent('openSubComment',{'detail': id}))
+          }, 500);
```

Comment:
Missing semicolon `;`
```suggestion
          window.setTimeout(() => {
            document.dispatchEvent(new CustomEvent('openSubComment',{'detail': id}));
          }, 500);
```
or 
```suggestion
          window.setTimeout(() => document.dispatchEvent(new CustomEvent('openSubComment',{'detail': id})), 500);
```